yb-voyager/cmd/analyzeSchema.go

```diff
@@ -619,22 +649,31 @@ func convertIssueInstanceToAnalyzeIssue(issueInstance queryissue.QueryIssue, fil
 		issueType = UNSUPPORTED_PLPGSQL_OBEJCTS
 	}
 
-	//TODO: how to different between same issue on differnt obejct types like ALTER/INDEX for not adding it ot invalid count map
-	increaseInvalidCount, ok := issueInstance.Details["INCREASE_INVALID_COUNT"]
-	if !ok || (increaseInvalidCount.(bool)) {
-		summaryMap[issueInstance.ObjectType].invalidCount[issueInstance.ObjectName] = true
+	var constraintIssues = []string{
+		queryissue.EXCLUSION_CONSTRAINTS,
+		queryissue.DEFERRABLE_CONSTRAINTS,
+		queryissue.PK_UK_ON_COMPLEX_DATATYPE,
 	}
+	displayObjectName := issueInstance.ObjectName
```

Comment:
I think  the displayName logic should live along with the Issue struct itself. For every issue type where we need a different display name, we will have to come here and write logic in the future, which beats the point of the Issue abstraction.

Reply:
Just add a TODO here with this comment. 

// unsupportedIndexIssue
//   ObjectType = INDEX
//   ObjectName = idx_name ON table_name
//   invalidCount.Type = INDEX
//   invalidCount.Name = ObjectName (because this is fully qualified)
//   DisplayName = ObjectName

// deferrableConstraintIssue
//  ObjectType = TABLE
//  ObjectName = table_name
// 	invalidCount.Type = TABLE
//  invalidCount.Name = ObjectName
//  DisplayName = table_name (constraint_name) (!= ObjectName)


// Solutions
// 1. Define a issue.ObjectDisplayName 
// 2. Keep it in issue.Details and write logic in UI layer to construct display name.